### ISSUE TITLE
The image helper now produces absolute paths

### ISSUE THREAD

**Issue by jixam** [mojolicious/mojo#2107]
* Mojolicious version: 9.34
* Perl version: 5.36
* Operating system: Ubuntu 22.04 LTS

### Steps to reproduce the behavior
```
perl -E 'use Mojolicious; say join ": ", $Mojolicious::VERSION, Mojolicious->new->build_controller->image("test.png")'
```

### Expected behavior
We noticed this change in behavior which does not seem to be mentioned in Changes.

Before we adjust our application we wanted to check that the change is intended?

### Actual behavior
Previous output:
```
9.33: <img src="test.png">
```

New output:
```
9.34: <img src="/test.png">
```

**Comment by kraih** [comment]
Yes, this is intentional. The generated URL is supposed to be relative to the static directory of the application.

**Comment by jixam** [comment]
All right, thank you for the clarification.

This is quite a breaking change for us as we have been treating the parameter as just the verbatim `src` attribute, doing things like `image("https://example.org/test.png")`. But we will cope :)

**Comment by kraih** [comment]
Absolute URLs are actually supposed to be passed through unchanged. If that doesn't work currently it would be incompatible with mojo.js and therefore a bug.

**Comment by rawleyfowler** [comment]
We're effected by this as well, I'll take a crack at a PR.

**Comment by rawleyfowler** [comment]
Here's an instance of the behavior in one of our projects:

```
src="/https://reasonml.github.io/img/reason.svg"
```

**Comment by sergiotarxz** [comment]
I am getting to work in this.

**Comment by sergiotarxz** [comment]
Should I change this behavior in sub _file_url?

**Comment by jixam** [comment]
I will go ahead and close this issue since it was mostly fixed by #2112.
